bower.json

```diff
@@ -1,10 +1,20 @@
 {
   "name": "cartodb.js",
-  "version": "0.0.1",
-  "dependencies": {},
-  "devDependencies": {
-    "backbone": "~1.1.2",
-    "modernizr": "~2.8.3",
-    "jquery": "~2.1.1"
-  }
+  "main": [
+    "cartodb.js",
+    "themes/css/cartodb.css"
+  ],
+  "version": "3.15.1",
```

Comment:
It would be nice if this number got automagically updated when someone runs `grunt build`. Right now [we update some versions in examples, README, etc.](https://github.com/CartoDB/cartodb.js/blob/develop/grunt/tasks/replace.js).